app/cli/main.cli.ts

```diff
@@ -55,4 +61,24 @@ export default class MainCli {
   getBreakdown(): boolean {
     return this.breakdown
   }
+
+  listHelpCommands(): Array<{ key: string; value: string }> {
```

Comment:
perhaps best moved into a service so its easier to test `CLIOutputService` or something.